pegasus/sites.v3/code.org/views/dance_tutorial_section.haml

```diff
@@ -1,3 +1,4 @@
+- hide_keep_on_dancing_tile = DCDO.get('hide_dance_followup', nil)
```

Comment:
nit: a default value of `'false` would be slightly more appropriate given we do a "truthiness" check with the `unless` below, but this is fine.